book/crud.rst

```diff
@@ -0,0 +1,767 @@
+        } %}
+        {{ parent() }}
+    {% endblock content_data %}
+
+The template extends the `update.html.twig`_ base template that is defined in the OroUIBundle. You
+just have to customize some important blocks to get desired output:
+
+``navButtons``
+
+    With this block, you can customize which buttons will be shown above the form allowing the user
+    to save the form and to trigger additional options. Beware that you set the ``formAction``
+    variable to a proper value depending on the controller the user accessed.
+
+``pageHeader``
+
+    This is the title block that is used to show a breadcrumb-style navigation and a custom title.
+
+``content_data``
+
+    This block defines the main contents of the update page which is read from the ``data``
+    variable by the parent template. If you want to divide your form into several sections, you can
+    define multiple ``dataBlocks`` here for this purpose.
+
+.. seealso::
+
+    Take a look at the templates that are shipped with the OroUIBundle to find out what blocks you
+    can also use to customize the layout of your forms.
+
+After the user submitted the form and the entered data is valid, the data will be persisted using
+the entity manager and a flash message is added which will be shown to the user on the request. The
+``oro_ui.router`` service is then used to redirect the user depending on the button they clicked to
+submit the form. They will be redirected to the first route that is passed to
+``redirectAfterSave()`` if they chose the *Save* button and if the user chose the *Save and close*
+button they will be redirected to the second route.
+
+.. _book-crud-view:
+
+Show Details of an Entity
+-------------------------
+
+To display all the details of an entity, you have to perform two steps. First, you need to create a
+controller that fetches the object from the database and passes it to a template:
+
+.. code-block:: php
+    :linenos:
+
+    // src/InventoryBundle/Controller/VehicleController.php
+    namespace InventoryBundle\Controller;
+
+    use InventoryBundle\Entity\Vehicle;
+    use Oro\Bundle\SecurityBundle\Annotation\AclAncestor;
+    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+    // ...
+
+    class VehicleController extends Controller
+    {
+        /**
+         * @Route("/{id}", name="inventory.vehicle_view", requirements={"id"="\d+"})
+         * @Template
+         * @AclAncestor("inventory.vehicle_view")
+         */
+        public function viewAction(Vehicle $vehicle)
+        {
+            return array('vehicle' => $vehicle);
+        }
+    }
+
+And of course you also have to create the matching template:
+
+.. code-block:: html+jinja
+    :linenos:
+
+    {# src/InventoryBundle/Resources/views/Vehicle/view.html.twig #}
+    {% extends 'OroUIBundle:actions:view.html.twig' %}
+    {% import 'OroUIBundle::macros.html.twig' as UI %}
+
+    {% block navButtons %}
+        {% if resource_granted('EDIT', vehicle) %}
```

Comment:
It's called `vehicle` in https://github.com/orocrm/documentation/pull/140/files#diff-2568e9fc0bcc3165392750dd6f915dd6R615. Or is there something in the UIBundle that requires the entity variable to be named `entity`?